homeassistant/components/zwave.py

```diff
@@ -165,6 +170,10 @@ def start_zwave(event):
         """ Called when Home Assistant starts up. """
         NETWORK.start()
 
+        polling_interval = config[DOMAIN].get(CONF_POLLING_INTERVAL, None)
+        if polling_interval is not None:
+            NETWORK.setPollInterval(polling_interval)
```

Comment:
Could you add a section on this to our [Z-Wave documentation](https://github.com/balloob/home-assistant.io/blob/master/source/_components/zwave.markdown)?